packages/paste-core/components/list/src/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {Text, NULLIFY_TEXT_PROPS} from '@twilio-paste/text';
+
+interface BaseListProps extends React.OlHTMLAttributes<''> {
+  className?: never;
+  style?: never;
+  as: 'ol' | 'ul';
+}
+
+const List: React.FC<BaseListProps> = ({as, children, ...props}) => {
+  return (
+    <Text
+      {...props}
+      {...NULLIFY_TEXT_PROPS}
```

Comment:
Nullify any sneaky style props people might be forcing through, or able to pass when not using Typescript.